pkg/apis/serving/v1beta1/predictor_model.go

```diff
@@ -121,11 +124,6 @@ func (m *ModelSpec) runtimeSupportsModel(runtimeName string, srSpec *v1alpha1.Se
 }
 
 func (m *ModelSpec) getModelTypeLabel() string {
-	if m.Runtime != nil {
-		// constrain placement to specific runtime
-		return "rt:" + *m.Runtime
-	}
-	// constrain placement based on model type
```

Comment:
@pvaneck why this change and the one below? This is the logic I was referring to in model-mesh case where we use the runtime for placement if it's provided.

Reply:
In the predictor reconciler, I had added the check for `Runtime` being explicitly given. If it is, then I perform the call to `GetServingRuntime` and do checks on what is retrieved. 

Since that runtime checking logic was added in the predictor reconciler, I shifted the focus of these functions to only be concerned with model type labels. Currently, these are used to verify a runtime supports a predictors model type which needed the runtime checking logic here to be removed (otherwise `RuntimeSupportsModel` would always return true because of the `rt:` labels.